synthetics-client/scheduler.py

```diff
@@ -0,0 +1,119 @@
+"""
+Copyright OpenSearch Contributors
+SPDX-License-Identifier: Apache-2.0
+"""
+
+import logging
+from ping import Ping
+from apscheduler.schedulers.background import BackgroundScheduler
+import warnings
+
+# In an unreleased patch, the deprecation warnings are fixed but for now it is harmless
+# These warnings just indicate that a feature the Pytz library provided is now no longer needed
+warnings.filterwarnings("ignore", message="The localize method is no longer necessary")
+
+import os
+import yaml
+import time
+
+# Scheduler reads yml files in ./suites and based on the schedules kept inside, 
+# pings the endpoints at the specified regular intervals
+class Scheduler:
+    def __init__(self, client):
+        self.client = client
+
+        self.read_configs()
+    
+    def read_configs(self):
+        self.create_synthetics_index()
+
+        logging.info('Starting scheduler')
```

Comment:
Be clear on the log text, there could be other schedulers running, better to indicate this is the scheduler for synthetics client

Reply:
Fixed